ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4835,4 +4835,22 @@ kong_apis:
       config.hour: "{{ medium_rate_limit_per_hour }}"
       config.limit_by: credential
     - name: request-size-limiting
-      config.allowed_payload_size: "{{ medium_request_size_limit }}"
\ No newline at end of file
+      config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: "channelSpecificTelemetryExhaust"
+    request_path: "{{ data_service_prefix }}/v3/dataset/get"
+    upstream_url: "{{ sunbird_analytics_api_base_url }}/dataset/get"
+    strip_request_path: true
+    plugins:
+     - name: jwt
+     - name: cors
+     - "{{ statsd_pulgin }}"
+     - name: acl
+      config.whitelist:
+        - 'dataAccess'
```

Comment:
I think we should recheck on the ACL for this API

Reply:
Can you please suggest on ACL to use for this API

Reply:
As this functionality is just to get the telemetry exhaust details and we have secrity enabled from portal end for this, it can go with dataAccess for now @sowmya-dixit